delira/data_loading/augmenter.py

```diff
@@ -0,0 +1,520 @@
+import multiprocessing
+from multiprocessing import connection as mpconnection
+from collections import Callable
+import abc
+import os
+import sys
+import numpy as np
+import random
+
+from delira.data_loading.sampler import AbstractSampler, BatchSampler
+from delira.data_loading.data_loader import DataLoader
+from delira import get_current_debug_mode
+
+
+class _WorkerProcess(multiprocessing.Process):
```

Comment:
Nit: I would prefer moving this behind `_ParallelAugmenter` where it is used and not at the beginning of this file.